src/includePackage.test.js

```diff
@@ -0,0 +1,101 @@
+import {includePackage, commonTags} from './index'
+
+test('given just a string, it looks for scripts at a default location', () => {
+
+  jest.mock(`./packages/foo/package-scripts.js`, () => ({
+    scripts: {
+      default: 'echo foo',
+    },
+  }), {virtual: true})
```

Comment:
Oooohhhh! `virtual: true` is something new to me! Cool! So does this mean you can mock files that don't exist? This is very useful. TIL!

Reply:
Yep! However a gotcha is they seem to stay alive for the entire lifetime of all the tests not just the scope of that one test